src/status_im/signing/core.cljs

```diff
@@ -202,13 +206,47 @@
     (when (and (not in-progress?) (seq queue))
       (show-sign cofx))))
 
+(fx/defn send-transaction-message
+  {:events [::send-transaction-message]}
+  [cofx chat-id transaction-hash signature]
+  {::json-rpc/call [{:method "shhext_sendTransaction"
+                     :params [chat-id transaction-hash (:result (types/json->clj signature))]
+                     :on-success #(re-frame/dispatch [:transport/message-sent % 1])}]})
```

Comment:
what does 1 mean ?

Reply:
1 means that only 1 confirmation is needed (as opposed to `n` for group chats), it's an artifact of when sending was mostly in status-react, we will remove it at some point but still some work to do